src/actions/instances.ts

```diff
@@ -315,7 +315,7 @@ export const clearParameterMidiMappingOnRemote = (id: InstanceStateRecord["id"],
 		if (!param) return;
 
 		const meta = param.getParsedMeta();
-		if (typeof meta === "object") {
+		if (typeof meta === "object" && !Array.isArray(meta)) {
```

Comment:
fixes the TS build error as `typeof []` is also an array. I guess the issue here is that `AnyJSON` means it could be any kind of value. I wonder if really what we want to ensure is that meta is a `Record<string, AnyJSON>` throughout and if not fall back to `{}`. Maybe even the Meta Editor should not allow any non _JsonMap_ input as currently one could also provide `[]` which is valid JSON

Reply:
We have meta that we support but a user might have other ideas for it.. if they use midi mapping, osc mapping, they need to use objects but if they're simply adding meta that they use in their own way, i figure we shouldn't limit them?

Reply:
Hm, sure but the `[param]` object doesn't support it either - in fact doing anything than a `Record<string, AnyJson>` throws. Additionally setting it to eg `[]` means that functionality like MIDI mapping won't work so I personally am in favor of enforcing that it needs to be a JSON map at the top level.